bin/mason-js

```diff
@@ -68,14 +70,28 @@ function run(args, callback){
           if (result){
             log.info(`${package} added to mason-versions.ini`);
           }
+          return callback(null, true);
         });
       }
+      return callback(null, true);
     });
   }
 }
 
 if (require.main === module) {
-  run(args, function(err, result){
-    if (err) return callback(err);
+var masonIni = path.join(process.cwd(), '/mason-versions.ini');
+   
+if (require.main === module) {
+  run(args, masonIni, function(err, result){
```

Comment:
Looks like this cli tool either:
- errors
- does the thing and logs success

So perhaps there is no need for returning a second arg `result` from `run()`, and the two end scenarios would then be:
- `return callback(err)` --> process.exit(1)
- `return callback()` --> process.exit(0)

Reply:
👍 just updated and remove result